apps/app/src/views/routes/Settings/components/templates/TemplateForm/utils.ts

```diff
@@ -9,7 +9,12 @@ export const TEMPLATE_SCHEMA = yup.object({
     })
     .required('Please select a treatment...'),
   dispenseAsWritten: yup.boolean().nullable().optional(),
-  dispenseQuantity: yup.number().min(1, 'Must be 1 or more...').default(1).nullable().optional(),
```

Comment:
the fact that this specific check leads me to believe that there might be some downstream impacts here - any concerns?

Reply:
we have it working like this in solidjs elements for months now, this was just incorrectly implemented in the react side